### DISCUSSION TITLE
Monthly `fastn` Dependencies Update

### DISCUSSION THREAD

**Discussion by Arpita-Jaiswal** [fastn-stack/fastn#1445] [General]
# Monthly `fastn` Dependencies Update

This discussion is for the purpose to keep track of monthly updates done for external dependencies that `fastn` uses. These dependencies doesn't include rust dependencies as this can be updated using `cargo update`.

The following are the current dependencies used by `fastn`:

| Dependency | Current Version | Available Version | CDN link |
|---|---|---|---|
| [Marked.js](https://github.com/markedjs/marked/) | 9.1.4 | [10.0.0](https://github.com/markedjs/marked/releases/tag/v10.0.0) | https://cdnjs.com/libraries/marked |
| [Prism.js](https://github.com/PrismJS/prism) (prism, bash, javascript, json, markdown, python, rust, sql, line-highlight, line-number) | 1.24.1 | 1.29.0 | https://cdnjs.com/libraries/prism |
| Cargo update | | | |

**Comment by amitu** [comment]
Future updates will happen in this way: https://github.com/fastn-stack/fastn.com/pull/513.
